helm/node-problem-detector/values.yaml

```diff
@@ -4,7 +4,9 @@ node-problem-detector:
 
   labels:
     application.giantswarm.io/team: phoenix
-
+  
+  used_space_threshold: 99
```

Comment:
Let's use camel case as suggested [in the docs](https://helm.sh/docs/chart_best_practices/values/#naming-conventions).

Reply:
done